examples/models/plot_aperiodic_params.py

```diff
@@ -0,0 +1,177 @@
+
+# Check the aperiodic parameters
+fm.aperiodic_params_
+
+###################################################################################################
+
+# Plot annotated model of aperiodic parameters
+plot_annotated_model(fm, annotate_peaks=False, annotate_aperiodic=True, plt_log=True)
+
+###################################################################################################
+# Comparing Offset & Exponent
+# ---------------------------
+#
+# A common analysis of model fit parameters includes examining and comparing changes
+# in the offset and/or exponent values of a set of models, which we will now explore.
+#
+# To do so, we will start by simulating a set of power spectra with different exponent values.
+#
+
+###################################################################################################
+
+# Define simulation parameters, stepping across different exponent values
+exp_steps = Stepper(0, 2, 0.25)
+ap_params = param_iter([1, exp_steps])
+
+###################################################################################################
+
+# Simulate a group of power spectra
+freqs, powers = gen_group_power_spectra(\
+    len(exp_steps), [3, 40], ap_params, [10, 0.25, 1], freq_res=0.25, f_rotation=10)
+
+###################################################################################################
+
+# Plot the set of example power spectra
+plot_spectra(freqs, powers, log_powers=True)
+
+###################################################################################################
+
+# Initialize a group mode object and parameterize the power spectra
+fg = FOOOFGroup()
+fg.fit(freqs, powers)
+
+###################################################################################################
+
+# Extract the aperiodic values of the model fits
+ap_values = fg.get_params('aperiodic')
+
+###################################################################################################
+
+# Plot the aperiodic parameters
+plot_aperiodic_params(fg.get_params('aperiodic'))
+
+###################################################################################################
+
+# Compute the correlation between the offset and exponent
+spearmanr(ap_values[0, :], ap_values[1, :])
+
+###################################################################################################
+#
+# What we see above matches the common finding that that the offset and exponent are
+# often highly correlated! This is because if you imagine a change in exponent as
+# the spectrum 'rotating' around some frequency value, then (almost) any change in exponent
+# has a corresponding change in offset value! If you note in the above, we actually specified
+# a rotation point around which the exponent changes.
+#
+# This can also be seen in this animation showing this effect across different rotation points:
+#
+# ![spectralrotation](https://raw.githubusercontent.com/fooof-tools/Visualizers/main/gifs/specrot.gif "specrot")
```

Comment:
When I built the site, this doesn't get rendered.

Reply:
Ooops, I did markdown instead of rst. Fixed!